Include/Python.h

```diff
@@ -55,6 +55,10 @@
 #  include <intrin.h>             // __readgsqword()
 #endif
 
+#if defined(Py_GIL_DISABLED) && defined(__MINGW32__)
+#  include <intrin.h>             // __readgsqword()
```

Comment:
This include is not just for `__readqsqword` right? depending on the architecture, it could mean different functions. Also, would it work to merge the `defined(_MSC_VER)` and `defined(__MINGW32__)` tests? (mainly, would it be fine in the future or is there a chance where the includes could diverge?)

Reply:
Merging those would work, but @zooba was against doing it in a single if/def and so I did it this way. The comment was taken from the line above since this new line has the exact same purpose. Happy to change it if you want to suggest an alternative.

Reply:
Oh no need for the change then! I don't have a better comment but I was a bit surprised with `__getReg(18)` and `__readfsdword` (I have NO idea of what is present in these files!). Are `__getReg` and `__readfsdword` part of this include? if so, I would add them to the comment (otherwise, it's fine).

Reply:
Let's keep the original comment. Those who know the file will probably know the rest (that way, we reduce the diff). Thanks for clarifying this!

Reply:
My view is it's fairly clear. Anyone looking through the code might search for __readgsqword and find the others, so the comment does its job of indicating why that include is necessary. I wouldn't complicate it by adding everything in the comment and leave it as it is above, but happy to take your guidance.